lms/templates/ux/reference/teams-base.html

```diff
@@ -155,11 +155,218 @@ <h1 class="page-title">Solar Power</h1>
     <div class="page-content">
```

Comment:
is this really the main content on this page?  if so, lets shout it loud and proud with an ```<main>``` element.  Unless of course there is already one on this page.  There is only room for one.